api/__init__.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import connexion
+
+from schematic import CONFIG
+
+# path to `config.yml` file as constant
+CONFIG_PATH = os.path.abspath(os.path.join(__file__, "../../config.yml"))
```

Comment:
I would make the CONFIG_PATH `schematic_config.yaml` a flask config variable. Then load the flask config in the create_app() factory function. Then read `flask.app.config['SCHEMATIC_CONFIG']` and use the path to pass to `CONFIG.load_config(CONFIG_PATH)`. It will help keep SCHEMATIC & the api separate down the line.

Reply:
Does this [commit](https://github.com/Sage-Bionetworks/schematic/pull/486/commits/18cdf4e3b3edca6dbc5c5dcc8efa6d3172c0e011) properly address this review comment @hweej?